edb/server/conn_pool/src/algo.rs

```diff
@@ -288,7 +295,8 @@ pub trait PoolAlgorithmDataBlock: PoolAlgorithmDataMetrics {
     /// and there are waiting elements; otherwise, returns `None`.
     fn hunger_score(&self, will_release: bool) -> Option<isize> {
         let waiting = self.count(MetricVariant::Waiting);
-        let connecting = self.count(MetricVariant::Connecting);
+        let connecting =
+            self.count(MetricVariant::Connecting) + self.count(MetricVariant::Reconnecting);
```

Comment:
This was missed in the previous reconnecting PR -- including here. Required rebalancing of constants.